linking to index html page and creating models to store playlists, so…

mahrous-amer opened the pull request:
…ngs, and user data

mahrous-amer committed:
linking to index html page and creating models to store playlists, songs, and user data

mahrous-amer committed:
update

vnealv commented on .env.dev:
```diff
@@ -1,5 +1,5 @@
 DEBUG=1
-SECRET_KEY=<put_key>
+SECRET_KEY=%$u!1k23rg#re5)qqhjlo$am18^ld)nticq4&3@_8$@w_&x&1*
```
never push keys like that :D make sure your env files are excluded from your PR

vnealv commented on app/Dockerfile.bck:
```diff
@@ -0,0 +1,69 @@
+###########
```
I dont think we need this file

mahrous-amer committed:
solved .env.dev

mahrous-amer committed:
new file:   app/grinder/api_wrapper.py
	modified:   app/grinder/models.py
	modified:   app/grinder/urls.py
	modified:   app/grinder/views.py
	new file:   app/grinder_api/__init__.py
	new file:   app/grinder_api/admin.py
	new file:   app/grinder_api/apps.py
	new file:   app/grinder_api/migrations/0001_initial.py
	new file:   app/grinder_api/migrations/__init__.py
	new file:   app/grinder_api/models.py
	new file:   app/grinder_api/tests.py
	new file:   app/grinder_api/utils.py
	new file:   app/grinder_api/views.py
	modified:   app/my_django/settings.py
	modified:   app/my_django/urls.py
	modified:   app/requirements.txt
	new file:   app/static/css/styles0.css
	new file:   app/templates/index2.html

vnealv commented on app/grinder/migrations/0001_initial.py:
```diff
@@ -0,0 +1,49 @@
+        migrations.CreateModel(
+            name='grinderuser',
+            fields=[
+                ('user_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to=settings.AUTH_USER_MODEL)),
```
maybe a better Primary key name here? like `id`

mahrous-amer committed:
resolving models and adding viewsets for the resolved models

mahrous-amer committed:
adding simple front-end html landing page and adjusting api_wrapper.py which will need to create a file in the directory /app/grinder_api/config.cfg including a title [SPOTIFY] also the file should contian CLIENT_ID=***** and CLIENT_SECRET=***** provided by spotify while you are there add a callback url in your app settings

mahrous-amer committed:
further enhancing the look of the initial landing page
